scripts/new_project.kts

```diff
@@ -41,6 +45,38 @@ object NewProject {
         }
     }
 
+    private fun renamePackageNameFolders() {
+        showMessage("=> 🔎 Rename the package folders...")
+        modules.forEach { module ->
+            val srcPath = projectPath + File.separator + module + File.separator + "src"
+            File(srcPath)
+                .walk()
+                .maxDepth(2)
+                .filter { it.isDirectory && it.name == "java" }
+                .forEach { javaDirectory ->
+                    val oldDirectory = File(
+                        javaDirectory, TEMPLATE_PACKAGE_NAME.replace(
+                            oldValue = DOT_SEPARATOR,
+                            newValue = File.separator
+                        )
+                    )
+                    val newDirectory = File(
+                        javaDirectory, packageName.replace(
+                            oldValue = DOT_SEPARATOR,
+                            newValue = File.separator
+                        )
+                    )
+
+                    val tempDirectory = File(javaDirectory, "temp_directory")
+                    oldDirectory.copyRecursively(tempDirectory)
```

Comment:
Can we execute `copyRecursively` in `copyFiles` method  or adapt parameter inside there 🤔  So we can know when error

Reply:
We can use the `copyFiles()` if we wish now, like`copyFiles(oldDirectory.absolutePath,tempDirectory.absolutePath)` isn't it? I think the current implementation is also acceptable since it is the native android api. cc @Tuubz @Wadeewee

Reply:
oh, we should use `copyFiles` method in this case. They are the same 😓 @kaungkhantsoe please help me refactor this in your PR 💘 Many thanks 💯